### PR TITLE
image component

### PR DESCRIPTION
**Pull request by devAmlan**
This is an Image component. Please review my PR.

### CODE REVIEW

File: library/component/image/image.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.responsive_image{
+    max-inline-size:100%;
+    block-size:auto;
+}
+.round_image{
+    max-inline-size:100%;
+    block-size:auto;
+    width:300px;
```
Comment (srishtigautam21) [inline library/component/image/image.css:8]: change them to rem

File: library/component/image/image.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    border:none;
+    border-radius:50%;
+}
+
+@media only screen and (max-width: 600px) {
+    .round_image{
+        max-inline-size:100%;
+        block-size:auto;
+        width:250px;
+        height:250px;
```
Comment (srishtigautam21) [inline library/component/image/image.css:19]: rem

File: library/component/image/image.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
+.responsive_image{
+    max-inline-size:100%;
+    block-size:auto;
+}
+.round_image{
+    max-inline-size:100%;
+    block-size:auto;
+    width:300px;
+    height:300px;
```
Comment (DivyanshSareen) [inline library/component/image/image.css:9]: can use `aspect-ratio: 1 / 1` to keep width and height equal

**Review by DivyanshSareen — APPROVED** [review]
Good to merge.

File: library/component/image/image.css (hunk 3 above)
Comment (devAmlan) [reply library/component/image/image.css:9]: yeah I will update that, thanks divyansh

**Review by srishtigautam21 — COMMENTED** [review]
Change px to rem

**Pull request merged**